src/SwiftPackage.ts

```diff
@@ -94,6 +94,24 @@ export class SwiftPackage implements PackageContents {
         return this.contents?.targets ?? [];
     }
 
+    get executableProducts(): Product[] {
+        return this.products.filter((product, index, array) => {
```

Comment:
We can simplify this to:

```typescript
return this.products.filter(product => product.type.executable !== undefined);
```

Same for the other properties.

Reply:
good point